code/datums/traits/negative.dm

```diff
@@ -154,3 +154,15 @@
 	else if(prob(0.5) && dumb_thing)
 		to_chat(H, "<span class='danger'>You think of a dumb thing you said a long time ago and scream internally.</span>")
 		dumb_thing = FALSE //only once per life
+
+/datum/trait/dnr
+	name = "DNR"
+	desc = "Your genetic makeup is unstable, so much so that after death, you can't be revived through most methods"
```

Comment:
If you plan on making this the hellbound thing I'd change this to reflect that you can't be brought back by any known means (aka anything outside of adminnery)